.github/workflows/cairomakie.yaml

```diff
@@ -22,7 +22,7 @@ concurrency:
 
 jobs:
   test:
-    name: CairoMakie Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    name: Julia ${{ matrix.version }}
```

Comment:
If I'm not mistaken this name is used in the reference image viewer to check which run is which.